lib/trino-hive-formats/src/main/java/io/trino/hive/formats/line/openxjson/OpenXJsonDeserializer.java

```diff
@@ -268,11 +271,10 @@ void decodeValue(Object jsonValue, BlockBuilder builder)
 
             try {
                 BIGINT.writeLong(builder, parseLong(jsonString.value()));
-                return;
             }
-            catch (NumberFormatException | ArithmeticException _) {
+            catch (NumberFormatException | ArithmeticException e) {
+                throw new TrinoException(BAD_DATA, "Error Parsing a column in the table: " + e.getMessage(), e);
```

Comment:
High level, in all of these decoders we should consider making the `invalidJson` case (e.g.: [line 268](https://github.com/trinodb/trino/pull/22513/files#diff-650fca78165689cfe93fd306e25d533795f4b0ccc4eebcbf4ee8a0beb20a3f45R268)) consistent with the exception thrown when parsing fails